linkml_runtime/utils/schemaview.py

```diff
@@ -1105,13 +1105,18 @@ def get_uri(self, element: Union[ElementName, Element], imports=True, expand=Fal
                     raise ValueError(f'Cannot find {e.from_schema} in schema_map')
             else:
                 schema = self.schema_map[self.in_schema(e.name)]
-            pfx = schema.default_prefix
             if use_element_type:
                 e_type = e.class_name.split("_",1)[0]  # for example "class_definition"
                 e_type_path = f"{e_type}/" 
             else:
                 e_type_path = ""
-            uri = f'{pfx}:{e_type_path}{e_name}'
+            pfx = schema.default_prefix
+            # To construct the uri we have to find out if the schema has a default_prefix 
+            # or if a pseudo "prefix" was derived from the schema id.
```

Comment:
`schema.id` becomes the prefix of a URI and then it's not a "pseudo prefix".

If you consider the term "prefix" _occupied_ by the meaning "CURIE prefix" and have therefore added the "pseudo", then I would simply avoid the term "prefix" in this line. What about `# or if default_prefix is using schema id to build up a URI` or so?